docker/Dockerfile

```diff
@@ -32,6 +32,9 @@ ADD . /build
 
 RUN ./deps/readies/bin/getpy2
 RUN ./system-setup.py
+RUN apt-get update -qq
```

Comment:
@chayim This won't work on OSes that are not Debian-like. We better have something going for us within system-sytem.py.